tools/openmldb_ops.py

```diff
@@ -205,7 +205,7 @@ def RecoverData(executor):
             return
         for name in tables:
             if not RecoverTable(executor, db, name).OK():
-                return
+                log.error(f"recover table failed. db {db} name {name}, check log for detail")
```

Comment:
f-string format is not supported in python2

Reply:
cvt to str format